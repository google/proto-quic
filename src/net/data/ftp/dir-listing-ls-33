total -9560322322989056
-rw-r--r-- 1   stadmin         stgroup 4310450 Nov 17 13:12 01643.001.862.TestPermission-DUP0001.zip
-rw-r--r-- 1   stadmin         stgroup 2496430080 Nov 13 14:46 I_Base_01_RSE_R720.iso
-rw-r--r-- 1   stadmin         stgroup 478576612 Nov 13 08:11 I_Base_01_RSE_R720.zip
