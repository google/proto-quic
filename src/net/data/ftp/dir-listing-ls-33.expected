-
01643.001.862.TestPermission-DUP0001.zip
4310450
1993
11
17
13
12

-
I_Base_01_RSE_R720.iso
2496430080
1994
11
13
14
46

-
I_Base_01_RSE_R720.zip
478576612
1994
11
13
8
11
